.github/labeler.yml

```diff
@@ -0,0 +1,23 @@
+# Add affects/core label to all changes to the core package; exclude changes to test files
+affects/core:
+  - any: [ '/*.go', '!/*_test.go']
+
+# Add affects/services label to all changes to the services directory
+affects/services:
+  - 'services/*'
+
+# Add affects/tests label when any test file is changed
+affects/tests:
+  - '**/*_test.go'
```

Comment:
Would this be sufficient to also match the test files in the root directory, or we may need another `- /*_test.go` item?